trust/trusts.go

```diff
@@ -16,7 +16,8 @@ import (
 	"github.com/docker/libtrust/trustgraph"
 )
 
-type TrustStore struct {
+// Store defines a TrustStore
```

Comment:
Stores trusted certificates and permissions which are used to verify the signature keys on manifests. This is being deprecated by the notary work.